cmd/gardener-extension-provider-aws/app/app.go

```diff
@@ -184,19 +184,17 @@ func NewControllerManagerCommand(ctx context.Context) *cobra.Command {
 			}
 			awscontrolplane.DefaultAddOptions.ShootWebhooks = shootWebhooks
 
-			if err := controllerSwitches.Completed().AddToManager(mgr); err != nil {
-				controllercmd.LogErrAndExit(err, "Could not add controllers to manager")
-			}
-
-			// Cleanup leaked ClusterRoles from worker controller.
-			// See https://github.com/gardener-attic/gardener-extensions/pull/378/files and https://github.com/gardener/gardener/issues/2144.
-			// TODO: This code can be removed in a future version.
+			// Update shoot webhook configuration in case the webhook server port has changed.
 			c, err := client.New(restOpts.Completed().Config, client.Options{})
 			if err != nil {
 				controllercmd.LogErrAndExit(err, "Error creating client for orphaned ClusterRole cleanup")
```

Comment:
Since `CleanupLeakedClusterRoles` has been deleted (I hope on purpose), the error message here is wrong
```suggestion
				controllercmd.LogErrAndExit(err, "Error creating client for startup tasks")
```